<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UIDesigner</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionExport_code"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionParse"/>
    <addaction name="actionDel"/>
   </widget>
   <widget class="QMenu" name="menuCCNode">
    <property name="title">
     <string>class</string>
    </property>
    <addaction name="actionCCSprite"/>
    <addaction name="actionCCLabelTTF"/>
    <addaction name="actionCCMenu"/>
    <addaction name="actionCContainer"/>
    <addaction name="actionCCScale9Sprite"/>
    <addaction name="actionCCProgressTimer"/>
    <addaction name="actionCCLabelAtlas"/>
    <addaction name="actionCCScrollView"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>setting</string>
    </property>
    <addaction name="actionResource"/>
    <addaction name="actionSave_path"/>
    <addaction name="actionLoadLanguage"/>
    <addaction name="actionRenameTools"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>view</string>
    </property>
    <widget class="QMenu" name="menuRatio">
     <property name="title">
      <string>ratio</string>
     </property>
     <addaction name="actionRatio25"/>
     <addaction name="actionRatio50"/>
     <addaction name="actionRatio75"/>
     <addaction name="actionRatio100"/>
     <addaction name="actionRatio150"/>
     <addaction name="actionRatio200"/>
    </widget>
    <addaction name="menuRatio"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuCCNode"/>
   <addaction name="menuView"/>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New File...</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As ...</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionParse">
   <property name="text">
    <string>paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCCSprite">
   <property name="text">
    <string>CCSprite</string>
   </property>
  </action>
  <action name="actionCCLabelTTF">
   <property name="text">
    <string>CCLabelTTF</string>
   </property>
  </action>
  <action name="actionCCMenu">
   <property name="text">
    <string>CCMenu</string>
   </property>
  </action>
  <action name="actionResource">
   <property name="text">
    <string>Resource Path...</string>
   </property>
  </action>
  <action name="actionSave_path">
   <property name="text">
    <string>Save path...</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="text">
    <string>delete</string>
   </property>
   <property name="toolTip">
    <string>delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionCContainer">
   <property name="text">
    <string>CContainer</string>
   </property>
  </action>
  <action name="actionRatio25">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>25%</string>
   </property>
  </action>
  <action name="actionRatio50">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>50%</string>
   </property>
  </action>
  <action name="actionRatio75">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>75%</string>
   </property>
  </action>
  <action name="actionRatio100">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
  </action>
  <action name="actionRatio150">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>150%</string>
   </property>
  </action>
  <action name="actionRatio200">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>200%</string>
   </property>
  </action>
  <action name="actionCCScale9Sprite">
   <property name="text">
    <string>CCScale9Sprite</string>
   </property>
  </action>
  <action name="actionCCProgressTimer">
   <property name="text">
    <string>CCProgressTimer</string>
   </property>
  </action>
  <action name="actionCCLabelAtlas">
   <property name="text">
    <string>CCLabelAtlas</string>
   </property>
  </action>
  <action name="actionCCScrollView">
   <property name="text">
    <string>CCScrollView</string>
   </property>
  </action>
  <action name="actionExport_code">
   <property name="text">
    <string>export code</string>
   </property>
  </action>
  <action name="actionRenameTools">
   <property name="text">
    <string>renameTools</string>
   </property>
  </action>
  <action name="actionLoadLanguage">
   <property name="text">
    <string>loadLanguage</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
